tests/specs/benchmarks/storagevar01-spec.k

```diff
@@ -3,15 +3,15 @@ requires "verification.k"
 module STORAGEVAR01-SPEC
   imports VERIFICATION
 
-  // fn-execute 
+  // fn-execute
   rule
     <k> (#execute => #halt) ~> _ </k>
     <exit-code> 1 </exit-code>
     <mode> NORMAL </mode>
     <schedule> BYZANTIUM </schedule>
```

Comment:
Please update this to ISTANBUL, which was part of the initial goal.

Reply:
Done.